Multisig/Data/Services/Notification Service/RegisterNotificationTokenRequest.swift

```diff
@@ -7,27 +7,58 @@
 //
 
 import Foundation
+import Web3
 
 struct RegisterNotificationTokenRequest: JSONRequest {
-    let uuid: String?
+    let uuid: String
     let safes: [String]
     let cloudMessagingToken: String
     let bundle: String
     let version: String
     let deviceType: String = "IOS"
     let buildNumber: String
+    let timestamp: String
```

Comment:
Timestamp must be optional because it is not required (https://docs.google.com/document/d/1RqIfZkqdS_TZ4IQVjeWUhNI1ZguD3NfpohSSRLVxB5Q/edit#heading=h.h09ziydayww)

Reply:
This is optional for the backend, but we can always send it. Or should I remove it?

Reply:
it is not meant to send the timestamp without the signature, please make it optional

Reply:
ok